src/auth.js

```diff
@@ -43,25 +43,45 @@ const generateCodeVerifier = () => {
   return codeVerifier
 }
 
-const getApiKey = (useTableauPassword = false) => {
+const storeRefreshToken = (refreshToken) => {
   if (window.localStorage) {
-    let apiKey = window.localStorage.getItem(apiTokenKey)
-    if (window.tableau && useTableauPassword) {
-      apiKey = window.tableau.password || apiKey
+    window.localStorage.setItem(refreshTokenKey, refreshToken)
+
+    if (window.tableau) {
+      window.tableau.password = refreshToken
     }
-    return apiKey
+    return refreshToken
   }
   return null
 }
 
-const storeApiKey = (key) => {
+const getAccessToken = async (useTableauPassword = false) => {
   if (window.localStorage) {
-    window.localStorage.setItem(apiTokenKey, key)
+    let refreshToken = window.localStorage.getItem(refreshTokenKey)
```

Comment:
Reuse `getRefreshToken()` instead of duplicating the logic in it?

Reply:
Good catch!, thanks.